src/developer/web-api/new-tracker.md

```diff
@@ -336,6 +336,37 @@ Tracker notes do not have their dedicated endpoint; they are exchanged as part o
 
 > One between `uid` or `username` field must be provided. If both are provided, only username is considered.
 
+### Program stage working lists
+
+If you want to make changes to a program stage working list, such as creating, updating, or deleting it, you'll have to utilize the endpoint called *programStageWorkingLists*. These lists are used in the Capture app to present predetermined working lists that are relevant a specific program stage in the tracker user interface.
+
+
+
```

Comment:
We probably do not have a standard to document enpoints and CRUD operations but I wouldn't understand how to call the API just from the description of the payload.
Maybe one example would make it more clear

Reply:
I added another paragraph as an introduction where I explain what is this resource for and what's the endpoint. Do you think this and the payload table would be clear enough for people to understand how to use it? Or would you still add an example?

Reply:
For me API docs like https://stripe.com/docs/api/balance are helpful in understanding the concepts and making the actual requests 😄 To me ideally there is a request I can copy and paste and run against the SL demo DB. Is the feature explained somewhere? If so can we link to it? Right now, I would not know what the feature is for.

We should probably take time to talk about our docs in general, maybe in the design meeting.

Reply:
The docs in the link you provided look great, but we are kind of far away from that at the moment, would be great if that was our goal though. For now, I extended the description of the endpoint (I hope it's clear), and added an example payload,